### PR TITLE
TPS-1083: Add deprecation notice to the docs

### PR DESCRIPTION
**Pull request by jonatascastro12**
https://zapierorg.atlassian.net/browse/TPS-1083

### CODE REVIEW

**Review by Jesswinters — DISMISSED** [review]

File: docs/_partner_api/endpoints.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ curl -H "Authorization: Bearer {token}" \
   -L "https://api.zapier.com/v1/zap-templates?client_id=${client_id}&limit=20&offset=20"
 ```
 
-## GET /v1/zap-templates/me
+## GET /v1/zap-templates/me [DEPRECATED ON JULY 5, 2021]
```
Comment (michaelgilley) [inline docs/_partner_api/endpoints.md:211]: I'd just delete this section so it's not in the docs at all.

Comment (michaelgilley) [reply docs/_partner_api/endpoints.md:211]: Docs for this endpoint are also on this page: https://platform.zapier.com/partners/zap-templates

https://github.com/zapier/visual-builder/blob/master/docs/_partners/zap-templates.md#my-zap-templates

**Review by michaelgilley — COMMENTED** [review]
Looking good @jonatascastro12. Just a couple of points.

File: docs/_partner_api/endpoints.md (hunk 1 above)
Comment (jonatascastro12) [reply docs/_partner_api/endpoints.md]: I removed those sections. 👍

**Review by AAorris — APPROVED** [review]

**Review by Jesswinters — APPROVED** [review]
✨ LGTM! [Love seeing those numbers](https://cdn.zappy.app/c10353cced5e3b5839839381fb598e02.png)

**Review by michaelgilley — COMMENTED** [review]
Thanks!

**Pull request merged**